app/views/layouts/application.html.slim

```diff
@@ -11,4 +11,6 @@ html
       = notice
     p.alert
       = alert
```

Comment:
what is "=alert"? above. And now you have added flash.each? remove line #13 probably

Reply:
that line was generated by default, removed